src/scripting/ScriptContext.cpp

```diff
@@ -117,7 +117,7 @@ ScriptContext::ScriptContext(LuaSandbox& aLuaSandbox, const std::filesystem::pat
 
     auto registerBinding = [this, &wrapHandler, &wrapDescription](
                                const std::string& acID, const std::string& acDisplayName, const std::variant<std::string, sol::function>& acDescription, sol::function aCallback,
-                               const bool acIsHotkey)
+                               const bool acIsHotkey, const bool acIsOverlayHotkey)
```

Comment:
I would rather have some enum with entries like 
* Input
* Hotkey
* OverlayHotkey
Instead of two weird bools when we have 3 options atm (even with two was a bit hacky choice by me back than...)